### PR TITLE
Adding the OutboundWebhook CRD and permissions to the helm chart

### PR DESCRIPTION
**Pull request by ankitdh7**
Changes:
- Added the `OutboundWebhook` crd to helm chart
- Updated the k8s clusterole permissions to support the outboundwebooks

Related Issue:

https://github.com/coralogix/coralogix-operator/issues/166

### CODE REVIEW

**Review by OrNovo — APPROVED** [review]
Thank you for your contribution!

**Comment by ankitdh7** [comment]
Hi @OrNovo Thanks for the review. I dont have write access to the repo and hence can't merge it. Could you please help me in merging it? Thank you!
image

**Pull request merged**
